web/test_cms.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/sh
```

Comment:
I'd like this file to live somewhere else -- maybe in /etc/ like you did with the test-waf.sh script.

Reply:
also, I think this will work, even with the '&':

for i in {1..70}
do
    curl -s -o /dev/null -w "[$(date +'%Y-%m-%d %H:%M:%S')] Attempt 70: I finished\n" localhost &
done